pkg/gotohelm/helmette/sprig.go

```diff
@@ -247,6 +247,13 @@ func MustRegexMatch(pattern, s string) bool {
 	return RegexMatch(pattern, s)
 }
 
+// MustRegexSplit is the go equivalent of sprig's `mustRegexMatch`.
+// +gotohelm:builtin=mustRegexSplit
+func MustRegexSplit(pattern, s string, n int) []string {
+	r := regexp.MustCompile(pattern)
+	return r.Split(s, n)
```

Comment:
On one hand split error is returned to the user, but on the other hand regexp can panic the execution. Could you use `regexp.Compile` and return an error too?

The same as it is in sprig
https://github.com/Masterminds/sprig/blob/8cb06fe3c8b0f1163c26b0a558669da72ee14656/regex.go#L73-L79

Reply:
Nice catch! I can never keep the meaning of `Must` straight. From gotohelm's perspective, they largely function the same.

Either way, I added support for both!